artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/Ubuntu-VM_OperatingSystemInterface-w1/source/src/main/resources/operatingSystemInterface.xsd

```diff
@@ -0,0 +1,72 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns="http://artifacttemplates.opentosca.org"
+           targetNamespace="http://artifacttemplates.opentosca.org">
+
+    <xs:element name="installPackageRequest">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element form="qualified" minOccurs="0" name="VMIP" type="xs:string"/>
```

Comment:
You can create a common type that provides all the shared properties, as in the [cloudProviderInterface.xsd](https://github.com/OpenTOSCA/tosca-definitions-common/blob/main/artifacttemplates/http%253A%252F%252Fopentosca.org%252Fartifacttemplates/OpenStack_CloudProviderInterfaceIA-w2/source/src/main/resources/cloudProviderInterface.xsd#L19)

Reply:
done